conf/messages.en

```diff
@@ -262,3 +268,18 @@ account.homePage.card.registration.details.2.body.partnership = View and change
 
 account.homePage.card.deregister.link = Deregister from Plastic Packaging Tax
 account.homePage.card.deregister.body = Cancel your Plastic Packaging Tax registration.
+
```

Comment:
are we putting place-hold for the welsh translation in the message.cy for these?